dash-spv-apple-bindings/src/coinjoin.rs

```diff
@@ -117,16 +107,15 @@ pub unsafe extern "C" fn add_client_queue_manager(
     context: *const c_void
 ) {
     let client_queue_manager = CoinJoinClientQueueManager::new(
-        Rc::new(RefCell::new(std::ptr::read(client_manager_ptr))),
```

Comment:
`std::ptr::read` creates a copy of client_manager, which is wrong. Using the raw pointer instead